03week/towersOfHanoi.js

```diff
@@ -19,24 +50,56 @@ function printStacks() {
   console.log("c: " + stacks.c);
 }
 
-function movePiece() {
-  // Your code here
-
+function movePiece(startStack, endStack) {
+  stacks[endStack].push(stacks[startStack].pop());
+  //moves the top stone from the start stack to the top of the end stack
 }
 
-function isLegal() {
-  // Your code here
-
+function isLegal(startStack, endStack) {
+  const startArr = stacks[startStack];
```

Comment:
good planning, just can be simplified. `return !endArr[0] || (startArr[startArr.length -1] < endArr[endArr.length -1]) `